services/horizon/cmd/root.go

```diff
@@ -142,6 +142,13 @@ var configOpts = support.ConfigOptions{
 		FlagDefault: uint(8000),
 		Usage:       "tcp port to listen on for http requests",
 	},
+	&support.ConfigOption{
+		Name:        "internal-port",
+		ConfigKey:   &config.InternalPort,
+		OptType:     types.Uint,
+		FlagDefault: uint(0),
+		Usage:       "WARNING: this should not be accesible from the Internet, tcp port to listen on for internal http requests, 0 (default) disables the internal server",
```

Comment:
@leighmcculloch good point! Does the following change look good?
```suggestion
		Usage:       "WARNING: this should not be accessible from the Internet and does not use TLS, tcp port to listen on for internal http requests, 0 (default) disables the internal server",
```

Reply:
👍